---a/net/minecraft/world/storage/IThreadedFileIO.java
+++b/net/minecraft/world/storage/IThreadedFileIO.java
@@ -1,7 +1,7 @@
 package net.minecraft.world.storage;
 
-public interface IThreadedFileIO
-{
+public interface IThreadedFileIO {
+
     boolean writeNextIO();
-}
+}
